### PR TITLE
Adds a note about the maximum version skew when upgrading dapr

### PR DESCRIPTION
**Pull request by JoshVanL**
In addition, please fill out the following to help reviewers understand this pull request:

## Description

I didn't previously find this written down in the docs so have made it explicit in the upgrade guide.

### CODE REVIEW

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-upgrade.md [hunk 1]
```diff
@@ -14,6 +14,12 @@ description: "Follow these steps to upgrade Dapr on Kubernetes and ensure a smoo
 ## Upgrade existing cluster to {{% dapr-latest-version long="true" %}}
 There are two ways to upgrade the Dapr control plane on a Kubernetes cluster using either the Dapr CLI or Helm.
 
+  *Note: Dapr only has a seamless guarantee when upgrading patch versions in a
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-upgrade.md]:
```suggestion
{{% alert title="Note" color="primary" %}}
Dapr only has a seamless guarantee when upgrading patch versions in a single minor version, or upgrading from one minor version to the next. For example, upgrading from `v1.6.0` to `v1.6.4` or `v1.6.4` to `v1.7.0` is guaranteed to be seamless. Upgrading more than one minor version at a time is best effort.
{{% /alert %}}
```

Just wrapping in shortcode alert and putting note on one line.

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-upgrade.md [hunk 2]
```diff
@@ -14,6 +14,12 @@ description: "Follow these steps to upgrade Dapr on Kubernetes and ensure a smoo
...
+  single minor version, or upgrading from one minor version to the next. For
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-upgrade.md]:
```suggestion
```

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-upgrade.md [hunk 3]
```diff
@@ -14,6 +14,12 @@ description: "Follow these steps to upgrade Dapr on Kubernetes and ensure a smoo
...
+  example, upgrading from `v1.6.0` to `v1.6.4` or `v1.6.4` to `v1.7.0` is
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-upgrade.md]:
```suggestion
```

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-upgrade.md [hunk 4]
```diff
@@ -14,6 +14,12 @@ description: "Follow these steps to upgrade Dapr on Kubernetes and ensure a smoo
...
+  guaranteed to be seamless.
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-upgrade.md]:
```suggestion
```

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-upgrade.md [hunk 5]
```diff
@@ -14,6 +14,12 @@ description: "Follow these steps to upgrade Dapr on Kubernetes and ensure a smoo
...
+  Upgrading more than one minor version at a time is best effort.*
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-upgrade.md]:
```suggestion
```

**Review by hhunter-ms — CHANGES_REQUESTED** [review]
Quick edit

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-upgrade.md [hunk 6]
```diff
@@ -14,6 +14,10 @@ description: "Follow these steps to upgrade Dapr on Kubernetes and ensure a smoo
 ## Upgrade existing cluster to {{% dapr-latest-version long="true" %}}
 There are two ways to upgrade the Dapr control plane on a Kubernetes cluster using either the Dapr CLI or Helm.
 
+{{% alert title="Note" color="primary" %}}
```
Comment (msfussell) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-upgrade.md:17]: I like this text, however it is best to move this to this section in the docs https://docs.dapr.io/operations/support/support-release-policy/#upgrade-paths that covers upgrades and then putting a reference link to this from this Kubernetes document. 

I would also change 
"guaranteed to be seamless" to "guaranteed tested"
 and 
 "Upgrading more than one minor version at a time is best effort" to "Upgrading more than one minor version at a time is untested and treated as best effort"

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-upgrade.md [hunk 7]
```diff
@@ -14,6 +14,10 @@ description: "Follow these steps to upgrade Dapr on Kubernetes and ensure a smoo
...
+Please refer to the [Dapr version policy]({{< ref "support-release-policy.md#upgrade-paths" >}}) for guidance on which versions of Dapr can be upgraded to which versions.
```
Comment (msfussell) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-upgrade.md]:
```suggestion
Refer to the [Dapr version policy]({{< ref "support-release-policy.md#upgrade-paths" >}}) for guidance on which versions of Dapr can be upgraded to which versions.
```

**Review by hhunter-ms — APPROVED** [review]

**Pull request merged**